index.js

```diff
@@ -390,7 +396,7 @@ module.exports = function (log, indexesPath) {
         }
     })
 
-    let offset = 0
+    let offset = -1
```

Comment:
I'm so sorry for going back and forth with this one, but maybe it would be better after all if it started with 0, and using the same pattern as in `updateIndex`, because there we needed to find the next possible offset (it may not be 0).

So perhaps `offset = 0`, and then using the same `if(!buffer) { offset++; return }` trick?

The concern I have is just consistency, that we use roughly the same patterns in `updateIndex` and `createIndexes`, to make it easier to read code, as well as avoid off by one errors with this -1 initialization.

Reply:
Alright :-)